src/ci/docker/armhf-gnu/Dockerfile

```diff
@@ -71,7 +71,8 @@ COPY scripts/qemu-bare-bones-addentropy.c /tmp/addentropy.c
 RUN arm-linux-gnueabihf-gcc addentropy.c -o rootfs/addentropy -static
 
 # TODO: What is this?!
```

Comment:
Do we need to explain what it is? In that case, it is Device Tree Blob of ARM Versatile Express Emulation 
for Cortex-A9.
See https://wiki.qemu.org/Documentation/Platforms/ARM or https://learningfromyoublog.wordpress.com/2016/04/05/131/ .